source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -34,6 +34,14 @@ For LabVIEW teams, there might be a few new :guilabel:`Third Party` items on var
 
 In order to install third party libraries for LabVIEW, download the VIs from the vendor (typically via some sort of installer). Then drag and drop the third party VIs into the respective folder mentioned above just like any other VI.
 
+How Does It Work? - Python
```

Comment:
Down below in the list of vendor deps, we should probably add references to the python package

Reply:
I agree that we should add references to the python packages, but it's likely that we could bikeshed it quite a bit. Let's do it in a future PR.